variables.tf

```diff
@@ -111,7 +111,7 @@ variable "existing_en_instance_crn" {
 
 variable "endpoint_type" {
   type        = string
-  description = "The endpoint type of the Secrets Manager instance required for establishing a connection between an Event Notifications service. Possible values are `public`, `private`."
+  description = "The type of endpoint to connect your Secrets Manager instance to Event Notifications. Possible values: public, private. The Terraform provider uses this value to configure notifications."
```

Comment:
I'm not sure this is clear still. This still seems to indicate the endpoint has something to do with secrets manager and Event Notifications communication. Its not.
@SirSpidey what we are trying to get across is:

- There is a Secrets Manager api that the IBM terraform provider calls to configure Event Notifications. That api can be called via the `public` or `private` secrets manager endpoint. This variable allows the user to pick which endpoint type to use.

Reply:
**New version**
The type of endpoint (`public` or `private`) to connect to the Secrets Manager API. The Terraform provider uses this endpoint type to interact with the Secrets Manager API and  configure Event Notifications.

Reply:
LGTM